### DISCUSSION TITLE
How does the creation of cards work?

### DISCUSSION THREAD

**Discussion by quijadajose** [anuraghazra/github-readme-stats#1934] [Q&A] [answered]
I found it interesting to be able to display GitHub stats dynamically and I'm curious what needs to be done to make this possible?

**Comment by rickstaa** [comment, marked as answer]
@UNDESC0N0CID0 Thanks for your question. Can you elaborate on what you mean by display dynamically? The cards are dynamically generated based on the current account stats (with a cache period) when you use the following markdown:

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
```

If you mean render based on the user theme, you can use something like this:

```html
<a href="https://www.linkedin.com/in/rickstaa#gh-light-mode-only">
<img src="https://img.shields.io/badge/LinkedIn-3572A5?style=for-the-badge&logo=linkedin&logoColor=white#gh-light-mode-only" alt="Connect with me on linkedin" >
</a>
<!-- Dark Mode -->
<a href="https://www.linkedin.com/in/rickstaa#gh-dark-mode-only">
<img src="https://img.shields.io/badge/LinkedIn-ffffff?style=for-the-badge&logo=linkedin&logoColor=0690FA#gh-dark-mode-only" alt="Connect with me on linkedin" >
</a>
```

For more information on dynamic themes, see #1415. If you want to know how we generate the cards, this is done using an `SVG` creation method (see https://github.com/anuraghazra/github-readme-stats/blob/master/src/common/Card.js). There is a proposal to release these common methods as an npm package so that other users can use them to create custom cards (see #1867).
